include/ibamr/IBFESurfaceMethod.h

```diff
@@ -110,7 +110,16 @@ class IBFESurfaceMethod : public IBStrategy
     static const std::string NORMAL_VELOCITY_SYSTEM_NAME;
     static const std::string PRESSURE_JUMP_SYSTEM_NAME;
     static const std::string TANGENTIAL_VELOCITY_SYSTEM_NAME;
+    static const boost::array<std::string, NDIM> VELOCITY_JUMP_SYSTEM_NAME;
     static const std::string VELOCITY_SYSTEM_NAME;
+    static const std::string WSS_O_SYSTEM_NAME;
+    static const std::string WSS_I_SYSTEM_NAME;
+    static const std::string P_O_SYSTEM_NAME;
+    static const std::string P_I_SYSTEM_NAME;
+    static const std::string TAU_SYSTEM_NAME;
+
+    SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > mask_var;
```

Comment:
Can this masking variable be removed from `IBFESurfaceMethod`?

Reply:
I think so. I used in the past but it's not needed any more in the current code.

Reply:
OK, let's remove it.

Reply:
I just realized I am actually using  mask_scratch_idx to synchronize the ghost data for the pressure at the IB.

https://github.com/IBAMR/IBAMR/blob/4d8bcdb9a61d8b84ecd8fd3d85331dbbd40e261a/src/IB/IBFESurfaceMethod.cpp#L533-L551

Reply:
OK, well make this an internal class variable and give it a name that corresponds to its use.

Reply:
(I should have said `private` rather than internal.)

Reply:
sounds good.